src/mds/Migrator.cc

```diff
@@ -219,12 +219,25 @@ void Migrator::find_stale_export_freeze()
    * - freeze subtree A
    * - client request tries authpinning items in subtree A
    *   (wait because subtree A is freezing)
+   *
+   *
+   *- For state EXPORT_WARNING:
+   *- When one or more CEPH_SESSION_FLUSHMSG or MSG_MDS_EXPORTDIRNOTIFY msgs
```

Comment:
From the code won't the `EXPORT_WARNING` state will wait for `MSG_MDS_EXPORTDIRNOTIFY` only ? How losing the `CEPH_SESSION_FLUSHMSG` could let the exporter be stuck in `EXPORT_WARNING` state ?